src/explore-education-statistics-admin/src/pages/publication/PublicationDetailsPage.tsx

```diff
@@ -0,0 +1,95 @@
+import PublicationDetailsForm, {
+  FormValues,
+} from '@admin/pages/publication/components/PublicationDetailsForm';
+import usePublicationContext from '@admin/pages/publication/contexts/PublicationContext';
+import publicationService from '@admin/services/publicationService';
+import themeService from '@admin/services/themeService';
+import Button from '@common/components/Button';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import SummaryList from '@common/components/SummaryList';
+import SummaryListItem from '@common/components/SummaryListItem';
+import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
+import useToggle from '@common/hooks/useToggle';
+import React from 'react';
+
+const PublicationDetailsPage = () => {
+  const { publication, onReload } = usePublicationContext();
+  const {
+    contact,
+    id,
+    permissions,
+    supersededById,
+    themeId,
+    title,
+    topicId,
+  } = publication;
+  const [readOnly, toggleReadOnly] = useToggle(true);
+
+  const { value, isLoading } = useAsyncHandledRetry(async () => {
+    const theme = await themeService.getTheme(themeId);
+    if (!supersededById) {
+      return { theme };
+    }
+
+    const supersedingPublication = await publicationService.getPublication(id);
+
+    return {
+      theme,
+      supersedingPublication,
+    };
+  });
+
+  const { theme, supersedingPublication } = value ?? {};
+
+  const topic = theme?.topics.find(themeTopic => themeTopic.id === topicId);
+
+  const handleSubmit = async (values: FormValues) => {
```

Comment:
Just a minor thing, but would export this as `PublicationDetailsFormValues` as other form components might also export their form values. It'd be good to be able to differentiate between exported form values.